bin/github.js

```diff
@@ -13,94 +10,110 @@ const octokit = new Octokit({
   auth: process.env.GITHUB_TOKEN,
 })
 
-async function getLatestRelease() {
-  const result = await octokit.repos.getLatestRelease({
-    owner: repoOwner,
-    repo: repository
-  })
-
-  return result.data
-}
-
-async function getTagByName(name) {
-  const perPage = 100
+class Github {
```

Comment:
Given the change to be a class now... I believe generate-release-notes.js will be broken as it has not been updated to use the class.